internal/usage/usage.go

```diff
@@ -39,6 +39,7 @@ const (
 	Forever                         = "Acknowledge an alert “forever”."
 	Status                          = "Alert's status."
 	Until                           = "Acknowledged until a date."
+	ConnectionStringType            = "When set to 'private' retrieves the private connection string. "
```

Comment:
[nit] let's be more specific here 
```suggestion
	ConnectionStringType            = "When set to 'private' retrieves the Network-peering-endpoint-aware connection string. "
```